operations/observability/mixins/workspace/rules/central/ws-manager.yaml

```
@@ -11,24 +11,14 @@ metadata:
   name: ws-manager-monitoring-rules
 spec:
   groups:
-  - name: ws-manager
-    rules:
-    - alert: GitpodWsManagerCrashLooping
-      labels:
-        severity: critical
-      annotations:
-        runbook_url: https://github.com/gitpod-io/runbooks/blob/main/runbooks/GitpodWsManagerCrashLooping.md
-        summary: Ws-manager is crashlooping in cluster {{ $labels.cluster }}.
-        description: Pod {{ $labels.namespace }}/{{ $labels.pod }} ({{ $labels.container }}) is restarting {{ printf "%.2f" $value }} times / 10 minutes.
-      expr: |
-        increase(kube_pod_container_status_restarts_total{container="ws-manager", cluster!~"ephemeral.*"}[10m]) > 0
-    - alert: GitpodWsManagerCrashLoopingMk2
-      labels:
-        severity: critical
-        dedicated: included
-      annotations:
-        runbook_url: https://github.com/gitpod-io/runbooks/blob/main/runbooks/GitpodWsManagerCrashLooping.md
-        summary: ws-manager-mk2 is crashlooping in cluster {{ $labels.cluster }}.
-        description: Pod {{ $labels.namespace }}/{{ $labels.pod }} ({{ $labels.container }}) is restarting {{ printf "%.2f" $value }} times / 10 minutes.
-      expr: |
-        increase(kube_pod_container_status_restarts_total{container="ws-manager-mk2", cluster!~"ephemeral.*"}[10m]) > 0
+    - name: ws-manager
+      rules:
+      - alert: GitpodWsManagerCrashLoopingMk2
+        labels:
+          severity: warning
+        annotations:
+          runbook_url: https://github.com/gitpod-io/runbooks/blob/main/runbooks/GitpodWsManagerCrashLooping.md
+          summary: ws-manager-mk2 is crashlooping in cluster {{ $labels.cluster }}.
+          description: Pod {{ $labels.namespace }}/{{ $labels.pod }} ({{ $labels.container }}) is restarting {{ printf "%.2f" $value }} times / 10 minutes.
+        expr: |
+          increase(kube_pod_container_status_restarts_total{container="ws-manager-mk2", cluster!~"ephemeral.*"}[10m]) > 0
```

could we keep it critical and change it to:
```suggestion
          increase(kube_pod_container_status_restarts_total{container="ws-manager-mk2", cluster!~"ephemeral.*"}[10m]) > 1
```
assuming when it restarts due to [WKS-210](https://linear.app/gitpod/issue/WKS-210) it only does once?
This way we should still get alerted if it was crashlooping

---

@WVerlaek the intent of this PR is to prevent noise for on-call.

`> 1` is too few, it'd still do false triggering given recent alerts for Dedicated cells in the last 7 days, and for gitpod.io in the last 30.

Check this out for my plan to improve critical alerting for gitpod.io and Dedicated: https://linear.app/gitpod/issue/WKS-277#comment-5dd289d8

---

In other words, you can expect a follow-on PR later today. As an interim solution, I just added this silence: https://alertmanager.gitpod.io/#/silences/0879f9b5-a44d-4db3-8667-f67a8c8440a1